themes/default/assets/sass/_container.scss

```diff
@@ -0,0 +1,19 @@
+/* purgecss start ignore */
```

Comment:
After a half-hour of trying to make this work in postcss.config.js, I gave up. This is valid, and works, so I decided to go with it.